### PR TITLE
Add "Create" button

### PR DESCRIPTION
**Pull request by Lenkelheit**
# Description

Added "Create" button.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Checklist:

- [x] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] My code builds clean without any errors or warnings

## Design

If you have change design fill in this part

### A picture of design before pull request
image

### A picture of design after pull request
image
image

### CODE REVIEW

File: Frontend/css/index.css [hunk 1]
```diff
@@ -34,3 +34,7 @@ body {
     text-align: center;
     border-top: 3px solid black;
 }
+
+.mg-10 {
+    margin: 10px;
```
Comment (iamprovidence) [inline Frontend/css/index.css]: do you really need this? I thought bootstrap already has it

File: Frontend/areas/administrator/index.html [hunk 2]
```diff
@@ -18,130 +18,145 @@
...
-                        <td>
-                            <input value="Delete" class="form-control" type="submit" name="Delete" form="change_user_form" />
-                        </td>
-                    </tr>
-
-
-
-                </tbody>
-            </table>
+            <div>
```
Comment (iamprovidence) [inline Frontend/areas/administrator/index.html]: wait.... you changed it only in **frontend** project? it takes me a 15 minutes to figure it out.

File: Frontend/css/index.css (hunk 1 above)
Comment (Lenkelheit) [reply Frontend/css/index.css]: yes

Comment (iamprovidence) [reply Frontend/css/index.css]: [Bootstrap Spacing](https://getbootstrap.com/docs/4.0/utilities/spacing/)

Comment (Lenkelheit) [reply Frontend/css/index.css]: > [Bootstrap Spacing](https://getbootstrap.com/docs/4.0/utilities/spacing/)

yes, of course

File: Frontend/areas/administrator/index.html (hunk 2 above)
Comment (Lenkelheit) [reply Frontend/areas/administrator/index.html]: yes, but I have added it to view just now.

**Pull request merged**